reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md

```diff
@@ -193,7 +193,7 @@ Accept wildcard characters: False
 ### -ScriptBlock
 
 Specifies the commands to run to perform tab completion. The script block you provide should return
-the values complete the input. The script block must unroll the values using the pipeline
+the values to complete the input. The script block must unroll the values using the pipeline
```

Comment:
I'm not 100% on the `to` here but without it, the sentence doesn't read right IMO.